src/main/java/de/pandemieduell/api/GamesController.java

```diff
@@ -0,0 +1,47 @@
+package de.pandemieduell.api;
+
+import de.pandemieduell.model.GameState;
+import de.pandemieduell.model.Round;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class GamesController {
+
+  @RequestMapping(value = "/registerUser", method = RequestMethod.POST, consumes = "text/plain")
+  public long registerUser(@RequestBody String username) {
+    return 0; // TODO implement
+  }
+
+  @RequestMapping(value = "/joinGame", method = RequestMethod.GET)
+  public long joinGame(@RequestParam("playerId") long playerId) {
```

Comment:
`GET` must always be indepodent! And since we are doing REST, we should talk about ressources, not actions. Why don’t we do something like `POST /games/{gameId}` to join an existing game?

Reply:
And `POST /games` for a new game